Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "api-dnsmasq-dynconf"
-version = "1.0.0"
+version = "1.1.0"
```

Comment:
Is this the right version increase[?](https://semver.org/)

Reply:
This is indeed incorrect versioning. I forgot to return a fitting http response code to justify a minor version increase.